Source/WebCore/Modules/notifications/Notification.cpp

```diff
@@ -426,6 +426,7 @@ void Notification::requestPermission(Document& document, RefPtr<NotificationPerm
 void Notification::eventListenersDidChange()
 {
     m_hasRelevantEventListener = hasEventListeners(eventNames().clickEvent)
+        || hasEventListeners(eventNames().auxclickEvent)
```

Comment:
This was in alphabetical order before, so maybe we could put `auxclickEvent` on the top line before the `||`?